Resources/Locale/en-US/paper/book-authorbooks.ftl

```diff
@@ -253,3 +253,12 @@ book-text-inspiration = As a freelance writer, Sarah was always in search of ins
       Without a moment's hesitation, Sarah took out her pen and notebook and began to write. She wrote about the beauty of the scene, the peacefulness she felt, and the surreal quality of the moment. She wanted to capture this feeling and share it with others through her words.
       As she finished writing, Sarah heard the snap of a twig and looked up to see a man approaching her with a camera. He introduced himself as a nature photographer and asked if she had seen the waterfall. They struck up a conversation, and soon they were both laughing and sharing stories of their adventures.
       As the sun began to set, the two of them packed up their gear and said their goodbyes. Sarah felt grateful for this unexpected encounter and the inspiration it brought her. She knew that this moment would stay with her forever, and she felt a sense of serenity knowing that there was still so much beauty in the world waiting to be discovered.
+
+book-text-janitor = [bold][color=black][head=3]The Tales of a Tired Janitor[/head][/bold]
+      Written by Caleb-Maxath
```

Comment:
Don’t reference your character.

Reply:
it's not my book, someone asked me to add it, will remove